Playbooks/Enrich-CIRCL-hashlookup/Playbook/azuredeploy.json

```diff
@@ -0,0 +1,514 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
metadata is completely missing. Check the below example for metadata fields,
https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/QualysVM/Playbooks/QualysVMPlaybooks/QualysVM-GetAssetDetails/azuredeploy.json

Reply:
fixed